src/app/routeValidator/helpers/validateRouteParameter.js

```diff
@@ -0,0 +1,12 @@
+import { throwError } from '../../dataValidator/utilities/messaging';
```

Comment:
Where is this actually being used? It seems you haven't integrated it with the actual article.jsx container. It's only in the article test.

Reply:
Line 8 of this helper file